styles.css

```diff
@@ -0,0 +1,138 @@
+*{
+    padding: 0;
+    margin:0;
+    box-sizing: border-box;
+}
+
+body{
+    font-family: 'Roboto', sans-serif;
+}
+
+/* Navigation bar */
+
+#navigation-bar{
+    height: 56px;
+    grid-template-columns: 0.5fr 1fr 0.5fr;
+    background-color: #ffffff;
+    padding: 16px 24px;
+}
+
+#logo{
+    width: 100px;
+    padding-left: 24px;
+}
+
+#searchbox{
+    padding: 5px 12px;
+    width: 85%;
+    border-radius: 3px 0px 0px 3px;
+    border: 1px solid hsl(0, 0%, 80%);
+    font-size: 1rem;
+}
+
+#searchbutton{
+    padding: 5px;
+    border-radius: 0px 3px 3px 0px;
+    margin-left: -1px;
+    border: 1px solid hsl(0, 0%, 80%);
+    width: 10%;
+    background-color: hsl(0, 0%, 97%);
+}
+
+#left-nav{
+    align-items: center;
+}
+
+#right-nav{
+    justify-self: end;
+}
+
+/* Main section */
+
+main{
+    grid-template-columns: 1fr 1fr;
+    /* margin-top: 20px; */
+    margin: 24px 34px;
+}
+
+/* Video-section */
+
+#video-section{
+    /* margin-left: 100px; */
+    /* margin-top: 20px; */
```

Comment:
Kindly remove all commented cod its a requirement at microverse since it keeps your code clean